docs/ios_xcuitest.md

```diff
@@ -86,5 +86,18 @@ xpaths("//some xpaths")
 - Specs by test code
     - https://github.com/appium/appium-xcuitest-driver/blob/master/test/functional/basic/gesture-e2e-specs.js
 
+### Workaround
+- `mobile:` commands depends on WDA and Apple's framework and the behaviour depends on them.
+- Sometimes issues occur such as "doubleTap isn't tapping #548"
+    - workaround for it:
+        ```ruby
+        def double_tap(element)
+          # Find the object, then go to the middle of it, and double-tap
+          tap_x = element.location.x + element.size.width * 0.50
+          tap_y = element.location.y + element.size.height * 0.50
+          execute_script 'mobile: doubleTap', x: tap_x, y: tap_y
```

Comment:
this code is not very effective, since it performs many extra calls to webdriver, which makes it longer, that it can be. More effective approach to minimise the count of Appium calls:

```ruby
rect = element.location.merge(element.size)
execute_script 'mobile: doubleTap', x: rect.x + rect.width / 2, y: rect.y + rect.height / 2
```

it might be even faster (only one WDA call instead of two) after selenium lib maintainers add **getRect** method to the _Element_ class, which is a part of Selenium 3+ standard and is already [available in Java lib](https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebElement.html#getRect--).